extensions/wikia/CreateNewWiki/tasks/ConfigureWikiFactory.php

```diff
@@ -85,34 +97,47 @@ public function getStaticVariables( $siteName, $imagesURL, $imagesDir, $dbName,
 			$wikiFactoryVariables[ 'wgWikiDirectedAtChildrenByFounder' ] = true;
 		}
 
+		// Language specific settings, moved here from wgLangCreationVariables
```

Comment:
Same situation as wgUniversalCreationVariables

Reply:
Please, refer to my previous comment. :)